crates/storage/provider/src/providers/static_file/writer.rs

```diff
@@ -547,6 +547,45 @@ impl StaticFileProviderRW {
         Ok(result)
     }
 
+    /// Appends multiple receipts to the static file.
+    ///
+    /// Returns the current [`TxNumber`] as seen in the static file.
+    pub fn append_receipts<I>(&mut self, receipts: I) -> ProviderResult<TxNumber>
+    where
+        I: IntoIterator<Item = Result<(TxNumber, Receipt), ProviderError>>,
+    {
+        let mut receipts_iter = receipts.into_iter().peekable();
+        // If receipts are empty, we can simply return the current TxNumber as seen in the static
+        // file.
+        if receipts_iter.peek().is_none() {
+            return Ok(self.writer.user_header().tx_end().expect("qed"));
```

Comment:
the CI is failing because it's not guaranteed to have at least one receipt in static files. Let's make this method return `Option<TxNumber>`.